src/ApiCore/BidiStream.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
- * Copyright 2016, Google Inc.
+ * Copyright 2017, Google Inc.
```

Comment:
I don't think we should update copyright years on existing source files, if all we do if modify them. This can be tricky because of the way git works - sometimes it thinks a new file is a modified old file, even though they are completely different. But in general if we make changes to an existing file, the copyright should not change.

Reply:
All set: https://github.com/googleapis/gax-php/pull/118/commits/30f9995614ee1481f21bb25c034fd5877721a701